nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+
+app.use("/numbers", router);
+
+router.get("/add", (req, res) => {
+    const firstNumber = req.query.first;
+    const secondNumber = req.query.second;
+    res.send(`Sum is ${Number(firstNumber) + Number(secondNumber)}`);
+});
+
+router.get("/multiply/:firstNum/:secondNum", (req, res) => {
+  res.send(`Result is ${Number(req.params.firstNum) * Number(req.params.secondNum)}`);
```

Comment:
Good use of both query parameters and request query.

Reply:
Thanks for your review. It'll help me with my homework for nodejs-week3 :)